src/openrct2-ui/windows/RideConstruction.cpp

```diff
@@ -2726,7 +2758,7 @@ static void WindowRideConstructionUpdateDisabledPieces(ObjectEntryIndex rideType
     const auto& rtd = GetRideTypeDescriptor(rideType);
     if (rtd.HasFlag(RIDE_TYPE_FLAG_HAS_TRACK))
     {
-        // Set all pieces as “disabled”. When looping over the ride entries,
+        // Set all pieces as �disabled�. When looping over the ride entries,
```

Comment:
These unicode chars keep getting munged and I don't know why. It's annoying to fix them, can we change them to regular bidirectional quotes?